client/sites/hooks/use-transfer-with-software-start-mutation.ts

```diff
@@ -0,0 +1,70 @@
+import { useMutation, UseMutationResult } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+
+type transferWithSoftwareResponse = {
+	transferId: number;
+};
+
+type SoftwareSlug = string;
+type SoftwareStatus = 'install' | 'activate';
+type Software = Record< SoftwareSlug, SoftwareStatus >;
+type ApiSettings = Record< string, unknown >;
+
+type transferOptions = {
```

Comment:
I think we should use PascalCase here.
```suggestion
type TransferOptions = {
```